src/common/curve_version.cpp

```diff
@@ -40,11 +40,29 @@ std::string CurveVersion() {
     return version;
 }
 
+std::string CurvefsVersion() {
+    static const std::string version =
+#ifdef CURVEFSVERSION
+#define STR(val) #val
+#define XSTR(val) STR(val)
+        std::string(XSTR(CURVEFSVERSION));
+#else
+        std::string("unknown");
+#endif
+    return version;
+}
+
 void ExposeCurveVersion() {
     static bvar::Status<std::string> version;
     version.expose_as("curve", "version");
     version.set_value(CurveVersion());
 }
 
+void ExposeCurvefsVersion() {
+    static bvar::Status<std::string> version;
+    version.expose_as("curvefs", "version");
+    version.set_value(CurvefsVersion());
+}
+
```

Comment:
why add another one?

Reply:
> 
Keep the difference from curve to distinguish